fabric-tool-attributes-v1/src/main/java/net/fabricmc/fabric/impl/tools/EntityToolActor.java

```diff
@@ -0,0 +1,17 @@
+package net.fabricmc.fabric.impl.tools;
+
+import net.fabricmc.fabric.api.tools.v1.ToolActor;
+import net.minecraft.entity.LivingEntity;
+
+public class EntityToolActor implements ToolActor<LivingEntity> {
```

Comment:
Make it a generic simple tool actor (like `BasicToolActor<T>`), or even better, make it a package-private class at the same package as `ToolActor`, and add a simple static method in `ToolActor` like `static <T> ToolActor<T> of(T actor)`